traefik/templates/_podtemplate.tpl

```diff
@@ -934,10 +934,18 @@
         {{- if .Values.deployment.additionalVolumes }}
           {{- toYaml .Values.deployment.additionalVolumes | nindent 8 }}
         {{- end }}
+        {{- $root := . }}
         {{- range $localPluginName, $localPlugin := .Values.experimental.localPlugins }}
```

Comment:
I find the ordering of volumes a bit weird looking at it now; probably the plugin-related volumes, i.e. this range and the `/plugins-storage` mount, should be grouped. However that is probably more of a follow-up thing to clean.

Reply:
Later on, I'm happy to work on this in a separate PR.